### PR TITLE
Re 153

### PR DESCRIPTION
**Pull request by cPholloway**
By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

**Comment by cPholloway** [comment]
The only difference between this and my RE-153 is that the `elevate-cpanel` file has been generated.  The rest of this is just from where this branch is behind main.

File: Changelog.md
Comment (toddr) [inline Changelog.md:1]: please do not include changelog in the PR.

File: version
Comment (toddr) [inline version:1]: please do not bump the version.

File: script/elevate-cpanel.PL [hunk 1]
```diff
@@ -782,6 +785,7 @@ sub run_service_and_notify ($self) {    # FIXME: move to Elevate::Service - need
         if ( $stage == VALID_STAGES ) {
             update_stage_file( { status => q[success] } );
             $self->_notify_success();
+            File::Copy::copy( ELEVATE_STAGE_FILE, ELEVATE_SUCCESS_FILE );
```
Comment (toddr) [inline script/elevate-cpanel.PL:788]: This looks like a revert?

**Review by toddr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**